samples/go/photo-dedup/job/dedupjob.go

```diff
@@ -6,26 +6,27 @@ package job
 
 import (
 	"fmt"
+	"runtime"
 
 	"github.com/attic-labs/noms/go/datas"
 	"github.com/attic-labs/noms/go/types"
+	"github.com/attic-labs/noms/go/util/status"
 	"github.com/attic-labs/noms/go/walk"
 	"github.com/attic-labs/noms/samples/go/photo-dedup/model"
-	"github.com/attic-labs/noms/go/util/status"
 )
 
 var grouper *photoGrouper
 
 // DeduplicateJob reads Set<Photo>'s (annotated with dhash) and writes Set<PhotoGroup> to
 // outDS where each group contains all duplicates.
-func DeduplicateJob(db datas.Database, photoSets []types.Value, outDS datas.Dataset) error {
-	return commitPhotoGroups(db, outDS, groupPhotos(db, photoSets))
+func DeduplicateJob(db datas.Database, photoSets []types.Value, outDS datas.Dataset, similarityThreshold int) error {
```

Comment:
"s...d" -> "threshold", or even "thresh" would be more go-ey.

you'll get used to it :)